ext/civicrm_admin_ui/ang/afsearchManageContributionPages.aff.json

```diff
@@ -0,0 +1,19 @@
+{
+    "type": "search",
+    "requires": [],
+    "entity_type": null,
+    "join_entity": null,
+    "title": "Manage Contribution Pages",
+    "description": "",
+    "is_dashlet": false,
+    "is_public": false,
+    "is_token": false,
+    "contact_summary": null,
+    "summary_contact_type": null,
+    "icon": "fa-list-alt",
+    "server_route": "civicrm/managecp",
```

Comment:
@aydun time to change the temp URL? I think this PR is mergeable. There are still issues but no blockers IMO.

Reply:
Changed & squashed